Framework/PythonInterface/mantid/api/src/Exports/IFunction.cpp

```diff
@@ -199,6 +213,23 @@ void export_IFunction() {
            "Split this function (if needed) into a list of "
            "independent functions")
 
+      .def("tie", &IFunction::tie,
+           (arg("self"), arg("parName"), arg("expr"), arg("isDefault")),
+           "Tie a parameter to an expression")
+
```

Comment:
`isDefault` is an optional argument of `IFunction::tie`. Shouldn't you use `BOOST_PYTHON_MEMBER_FUNCTION_OVERLOADS` in this case?